src/app/pages/student-display/student-display.component.html

```diff
@@ -1,35 +1,35 @@
 <ng-container [ngSwitch]="currentScreen$ | async">
-  <ng-container *ngSwitchCase="ScreenId.SessionLobby">
+
+  <app-student-round *ngSwitchCase="ScreenId.StudentRound"></app-student-round>
+
+  <ng-container *ngSwitchDefault>
     <app-hill-background cover></app-hill-background>
-    <div class="centered-container">
-      <mat-card>
+    <div class="centered-container" [ngSwitch]="currentScreen$ | async">
+
+      <mat-card *ngSwitchCase="ScreenId.SessionLobby">
         <mat-card-title>Waiting for round to begin&hellip;</mat-card-title>
+        <mat-card-content>
+          Please wait for the host to start a round.
+        </mat-card-content>
+        <mat-card-actions align="end">
+          <button mat-button routerLink="/">Exit Session</button>
```

Comment:
And then, just checking to make sure—we don't want this button to remove the student from the session in Firebase, do we?

Reply:
We don't currently have that implemented so I think that would be best left for another PR in the future if it is something we want.

Currently the component calls `sessionService.leaveSession()` when destroyed so if in the future we wanted that to be the case we could implement it there maybe.